docs/sdk/push-notifications/fragments/ios/getting-started.md

```diff
@@ -61,105 +68,112 @@ Use the following steps to connect push notification backend services to your ap
     import AWSPinpoint
     ```
 
-6. In the AppDelegate file, inside `application:didFinishLaunchingWithOptions:`, initialize AWS Pinpoint instance and register for push notifications from the user. When the app runs, the user will be prompt with a modal to allow notifications. We recommend you request for authorization from the user during app startup, so your users can begin receiving notifications as early as possible.
-
-```swift
-class AppDelegate: UIResponder, UIApplicationDelegate {
-
-    var pinpoint: AWSPinpoint?
+6. In the AppDelegate file, inside [`application(_:didFinishLaunchingWithOptions:)`](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622921-application), initialize AWS Pinpoint instance and register for push notifications from the user. When the app runs, the user will be prompt with a modal to allow notifications. We recommend you request for authorization from the user during app startup, so your users can begin receiving notifications as early as possible.
```

Comment:
Maybe I'm a stickler for this, but we should refer to services properly for a host of reasons. This sentence makes it seems like the service is called "AWS Pinpoint."

Suggestion: `AWSPinpoint`

Reply:
> Maybe I'm a stickler for this

haha not at all, I appreciate the careful review tbh. You're right, the intention here is to refer to the client class, so I fixed it and linked it to the actual documentation o the referred class. https://aws-amplify.github.io/aws-sdk-ios/docs/reference/AWSPinpoint/Classes/AWSPinpoint.html